src/api/firebase.js

```diff
@@ -55,6 +54,72 @@ export function getItemData(snapshot) {
 		.filter((item) => item.name !== null);
 }
 
+function sortItems(item1, item2) {
```

Comment:
This is technically a "compare" function as no sorting happens, just comparing two values and is passed as the compare function to a sort